samples/azure/blob-storage-databus/ABSDataBus_1/Receiver/Program.cs

```diff
@@ -16,7 +16,8 @@ static void Main()
         endpointConfiguration.UseSerialization<JsonSerializer>();
         var dataBus = endpointConfiguration.UseDataBus<AzureDataBus>();
         dataBus.ConnectionString("UseDevelopmentStorage=true");
-        endpointConfiguration.UsePersistence<InMemoryPersistence>();
+        endpointConfiguration.UseTransport<LearningTransport>();
+        endpointConfiguration.UsePersistence<LearningPersistence>();
```

Comment:
Is UsePersistence mandatory here? this receiver doesn't seem to use persistance, there are no saga's or does the  learning transport require a persister?